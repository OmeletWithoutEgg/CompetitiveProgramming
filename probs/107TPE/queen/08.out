4610929.04 -29738893.67
