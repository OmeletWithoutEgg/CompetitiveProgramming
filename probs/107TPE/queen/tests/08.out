-8259434.92 56857180.54
